resources/js/Pages/Home.vue

```diff
@@ -197,21 +197,5 @@ function showRandom(): void{
     .form-buttons {
         text-align: end;
     }
-
-    .cdx-field__control {
```

Comment:
Removing this from `Home.vue` because it was duplicated. This code belongs in `[ItemIdSearchTextarea.vue](https://github.com/wmde/wikidata-mismatch-finder/pull/831/files#diff-ee8902c55d2443db0d269e1a35909457d37b48d775e00e61fbc631dec2b8a759)` and has already been added there.